spec/rspec/expectations/configuration_spec.rb

```diff
@@ -87,6 +87,20 @@ def delegated?; true; end
         end
       end
 
+      describe "#max_formatted_output_length=" do
+        let(:object_with_large_inspect_string) { Struct.new(:value).new("a"*300) }
+
+        it "sets the maximum object formatter length" do
+          config.max_formatted_output_length = 10
```

Comment:
This needs to be restored after it's changed, or otherwise isolated, as it's currently leaking and breaking the build :(